### PR TITLE
LF-33 notification link

### PR DESCRIPTION
**Pull request by jeffersondarcy**
Notification is now clickable and navigation resolves internal and external routes

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -12,22 +12,25 @@ import { LernfairProvider } from './hooks/useLernfair';
 import { IssueReporter } from './IssueReporter';
 import { NotificationsProvider } from './components/NotificationsProvider';
 import { ToastNotifications } from './components/ToastNotifications';
+import { BrowserRouter } from 'react-router-dom';
```
Comment (timidak) [inline src/App.tsx:15]: This will clash with React Native. Iirc @Jonasdoubleyou mentioned that in the first meetings

Comment (timidak) [reply src/App.tsx:15]: Nevermind, I just saw that the BrowserRouter was already implemented 😄

Comment (Jonasdoubleyou) [reply src/App.tsx:15]: Yeah, c.f. https://github.com/corona-school/project-user/issues/552 ... Maybe we'll never build a full native app with react native, just a "more native PWA" ... Still, I wanted to keep the option open to somewhen go native without too much effort, so the usage of BrowserRouter here is fine ...

**Review by Jonasdoubleyou — APPROVED** [review]

File: src/components/notifications/MessageBox.tsx [hunk 2]
```diff
@@ -1,65 +1,91 @@
...
+    const navigateToLink = () => {
+        if (typeof navigateTo !== 'string') return null;
+        if (navigateTo.charAt(0) === '/') {
+            return navigate(navigateTo);
+        }
+        return navigateExternal();
+    };
 
-export default MessageBox
+    const navigateExternal = () => window.open(navigateTo, '_blank');
```
Comment (timidak) [inline src/components/notifications/MessageBox.tsx:42]: I'm not sure how a PWA would react on this event. But it should work for now.

**Review by timidak — APPROVED** [review]
LGTM 👍🏼

File: src/components/notifications/MessageBox.tsx (hunk 2 above)
Comment (Jonasdoubleyou) [reply src/components/notifications/MessageBox.tsx:42]: @timidak we can worry about the PWA / native topics once the User App is live and has notifications, appointments, ... ;)

**Pull request merged**